climpred/smoothing.py

```diff
@@ -0,0 +1,157 @@
+import xarray as xr
+
+try:
```

Comment:
safety measure to not get import errors

Reply:
Feels awkward to put a whole function in a try/except

I think it's better if you do a 

try:
    import xesmf as xe
except ImportError:
    xe = None


Then in the function

if xe is None:
   raise ImportError('xesmf is not installed; see https://xesmf.readthedocs.io/en/latest/installation.html')

Reply:
I think it's generally bad form to not import at the top, but this makes sense here. We don't want to do the try/except in the header and throw errors if they're not trying to coarsen. But I'll also add `xesmf` to the conda-forge requirements.